SourceKitStressTester/Sources/StressTester/StressTester.swift

```diff
@@ -53,7 +53,13 @@ public class StressTester {
       if let timingsFile = options.requestDurationsOutputFile {
         // We are only keeping track of code completion durations for now.
         // This could easily be expanded to other request types.
-        let timing = AggregatedRequestDurations(instructionCounts: codeCompletionDurations.map({ $0.instructionCount }))
+        let timings: [Timing] = codeCompletionDurations.compactMap({ (request, instructionCount) in
+          guard case .codeComplete(document: let document, offset: let offset, args: _) = request else {
+            return nil
+          }
+          return Timing(modification: document.modificationSummaryCode, offset: offset, instructions: instructionCount)
+        })
+        let timing = AggregatedRequestDurations(timings: timings)
```

Comment:
Nitpick: this seems more like it should be "durations" rather than "timing" but 🤷

Reply:
I’ll update it once I get rid of the “aggregated” bit of `AggregatedRequestDurations`